### DISCUSSION TITLE
[Question]: conversationsCache Doesn't seem to be used?

### DISCUSSION THREAD

**Discussion by kuangxiaoye** [danny-avila/LibreChat#1094] [Q&A]
### What is your question?

In ChatGPTClient.js 

sendMessage() doesn't seem to be called anywhere , so 'await this.conversationsCache.get / set 'not used either...

### More Details

Can you help me explain this? I now need to make the redis system work correctly, but the current project seems to be directly operating the mongodb database.

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by danny-avila** [comment]
The conversation cache for ChatGPTClient is not wired to use redis.

can you explain your use case as I’m not sure what you’re trying to do so I don’t know how to help

**Reply by kuangxiaoye** [reply]
Querying large amounts of conversation data directly in mongodb will cause performance issues....

So I was wondering if I could give priority to reading redis data when querying. When I was planning to start this matter, I found that the Free Dialogue project itself did not do this work. I don’t know what the considerations were?
